OneBusAway/en.lproj/Localizable.strings

```diff
@@ -475,14 +475,20 @@
 "msg_explanatory_google_analytics" = "Some information about how you use this app is sent to Google Analytics in non-personally identifiable form to help us better understand how to improve the app. To learn more, please read our Privacy Policy.";
 
 /* App feedback explanation on the Settings view controller. */
-"msg_explanatory_allow_app_feedback" = "You can also submit feedback in the Contact Us section on the previous page";
+"msg_explanatory_allow_app_feedback" = "You can also submit feedback in the Contact Us section on the previous page.";
 
 /* No comment provided by engineer. */
 "msg_something_went_wrong" = "Something Went Wrong";
 
 /* Error message displayed to the user when something goes wrong with a just-tapped shared trip. */
 "text_error_cant_show_shared_trip_param" = "Sorry, we can't show you that shared trip. If this continues, please let us know. Error: %@";
 
+/* Error message title displayed to the user when handoff failed to work. */
+"msg_handoff_failure_title" = "Uh oh! Handoff didn't work properly 😭";
+
+/* Error message body displayed to the user when handoff regions did not match both clients. */
+"msg_handoff_region_mismatch_body" = "Try setting your region to \"Automatically Select Region\" to fix this issue.";
```

Comment:
"Try enabling the \"Automatically Set Region\" option from the Regions section under the Info tab to fix this issue, and let us know if that doesn't work."